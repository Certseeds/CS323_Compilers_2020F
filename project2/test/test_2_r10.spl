struct Person
{
  int name;
  int friends[10];
};
int test_2_r10()
{
  struct Person tom;
  struct Person people[10];
  int i = 0;
  while (i < 10)
  {
    people[i].name = i;
    tom.friends[i] = i;
    i = i + 1;
  }
  return tom.name[i-1];
}
