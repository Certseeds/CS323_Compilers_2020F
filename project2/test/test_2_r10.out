Error type 10 at Line 17: indexing on non-array variable
